utils/utils.py

```diff
@@ -17,3 +19,80 @@ def preprocess_data(X, y=None, remove_zerovar=True, standardize=True):
             X = StandardScaler().fit_transform(X)
 
     return X, y
+
+
+def prox_l1sorted(v, w):
```

Comment:
try to harmonize notation: the other prox function takes `beta, lambdas` as input